src/library/Button/__tests__/Button.spec.js

```diff
@@ -1,7 +1,9 @@
 /* @flow */
 import React from 'react';
+import { BrowserRouter, Link as ReactRouterLink } from 'react-router-dom';
```

Comment:
Can we simulate a 3rd party component, without actually relying on the dependency?